x/auth/migrations/v2/store_test.go

```diff
@@ -6,6 +6,8 @@ import (
 	"testing"
 	"time"
 
+	"github.com/cosmos/cosmos-sdk/x/auth/ante"
```

Comment:
Careful here as well with the imports, make sure you're importing your local auth and not upstream sdk.

Reply:
Just for clarity, you only mean here to have a proper `replace` directive in the `go.mod`, right?